__tests__/index.test.ts

```diff
@@ -99,17 +118,81 @@ describe('Test main action', () => {
         expect(core.exportVariable).toHaveBeenCalledWith(ENV_NAME_3, SECRET_3);
         expect(core.exportVariable).toHaveBeenCalledWith(ENV_NAME_4, SECRET_4);
 
-        expect(core.exportVariable).toHaveBeenCalledWith(CLEANUP_NAME, JSON.stringify(['TEST_ONE_USER', 'TEST_ONE_PASSWORD', 'TEST_TWO_USER', 'TEST_TWO_PASSWORD', ENV_NAME_3, ENV_NAME_4]));
+        // Case when alias is blank, but still comma delimited in workflow and json is parsed
+        // ex: ,test5/secret
+        expect(core.exportVariable).toHaveBeenCalledWith("_USER", "integ");
```

Comment:
Can we implement it in way that it doesn't have leading `_`? Is there some usability reason for these not being:
- `USER`
- `PASSWORD`
- `CONFIG_ID1`

Reply:
I wouldn't say there's a usability reason, it's just that the natural behavior of the recursive parsing function is to introduce a leading underscore so I felt it was more logically consistent that way. But it's definitely not a problem to change.

Essentially the idea is to support the empty string (or trimmed whitespace) as a "prefix" which means that the underscore is still the second character in a manner of speaking.